README.md

```diff
@@ -1 +1,62 @@
-# gluster-ansible-maintenance
\ No newline at end of file
+Role Name
+=========
+
+replace_node - Replace gluster node with a new node.
+
+Requirements
+------------
+
+- Ansible >= 2.6
+- lvm2 (Optional)
+
+
+NOTE: Inventory should contain a single node, which should be part of the cluster. This node should be different than any of the three variables explained below, this requirement is to collect the right peers for the new node.
+
+Role Variables
```

Comment:
Can we have a Requires column as well in the table?

Reply:
Ack! Sorry for missing this.

Reply:
Ack! done.